test/TesterInternal/Serialization/BuiltInSerializerTests.cs

```diff
@@ -65,8 +59,7 @@ public BuiltInSerializerTests(ITestOutputHelper output)
         }
 
         [Theory, TestCategory("Functional"), TestCategory("Serialization")]
-        [InlineData(SerializerToUse.Default)]
```

Comment:
It makes it easier to add & test other serializers (eg, `IlBasedSerializer`) without modifying every method.

In the (legitimate?) case where one serializer is intended to fail and another is intended to succeed, then we can use `InlineData` to specify that.

I'm happy to revert the change and simply remove the do-nothing `Fallback` option.